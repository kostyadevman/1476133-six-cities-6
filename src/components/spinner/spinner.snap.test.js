import React from 'react';
import {render} from '@testing-library/react';
import Spinner from "./spinner";

it(`Should Spinner render correctly`, () => {
  const {container} = render(<Spinner />);
  expect(container).toMatchSnapshot();
});
