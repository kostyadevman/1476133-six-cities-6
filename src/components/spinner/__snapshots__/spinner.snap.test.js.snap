// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Spinner render correctly 1`] = `
<div>
  <p>
    Loading ...
  </p>
</div>
`;
